src/Backend/Core/Engine/Base/Action.php

```diff
@@ -128,7 +128,7 @@ public function display($template = null)
          * based on the name of the current action
          */
         if ($template === null) {
-            $template = $this->getBackendModulePath() . '/Layout/Templates/' . $this->URL->getAction() . '.tpl';
+            $template = '/'. $this->getModule() . '/Layout/Templates/' . $this->URL->getAction() . '.html.twig';
```

Comment:
this one will be fixed after merging https://github.com/Katrienvh/forkcms/pull/5